docs/getting-started-single-cluster.md

```diff
@@ -54,7 +59,7 @@ Run the following:
 ```bash
 curl "https://raw.githubusercontent.com/kuadrant/kuadrant-operator/${KUADRANT_REF}/hack/quickstart-setup.sh" | bash
 ```
-This will setup a single kind cluster, install Istio via the `project sail` operator and install Kuadrant. Once this completes you should be able to move on to using the various policy apis offered by Kuadrant
+This will setup a single kind cluster, install Istio and install Kuadrant. Once this completes you should be able to move on to using the various policy apis offered by Kuadrant
```

Comment:
```suggestion
This will setup a single kind cluster, install Istio and install Kuadrant. Once this completes you should be able to move on to using the various policy apis offered by Kuadrant. To use `project sail` to install istio, ensure you set the  `ISTIO_INSTALL_SAIL=true` env var before executing the quick start. 
```

Reply:
Done, thanks